src/main/java/organice/logic/parser/AddressBookParser.java

```diff
@@ -58,6 +58,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearCommand.COMMAND_WORD:
             return new ClearCommand();
 
+        case FuzzyFindCommand.COMMAND_WORD:
+            return new FuzzyFindCommandParser().parse(arguments);
+
         case FindCommand.COMMAND_WORD:
```

Comment:
Just wondering if you would like to rename 'FindCommand' to 'ExactFindCommand' for clarity.